.circleci/config.yml

```
@@ -15,8 +15,51 @@ jobs:
       - run:
           name: Run tests
           command: python -m pytest
+  glean-source:
+    executor:
+      name: python/default
+      tag: 3.10.10
+    steps:
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+      - &set_env 
+        run:
+          name: "Set environment variables"
+          command: echo 'export DATAHUB_GMS_URL="https://mozilla.acryl.io/gms"' >> "$BASH_ENV"
+      - run:
+          name: Sync Glean Pings
+          command: datahub ingest -c recipes/glean_recipe.dhub.yaml
+  legacytelemetry-source:
+    executor:
+      name: python/default
+      tag: 3.10.10
+    steps:
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+      - *set_env
+      - run:
+          name: Sync Legacy Telemetry Pings
+          command: datahub ingest -c recipes/legacy_recipe.dhub.yaml
 
 workflows:
   ci:
     jobs:
       - unit-tests
+  nightly:  # Sync custom integration sources
+    jobs:
+      - unit-tests
+      - glean-source:
+          requires:
+            - unit-tests
+      - legacytelemetry-source:
+          requires:
+            - unit-tests
```

Modify the jobs instances to include the new parameter

```suggestion
  nightly:  # Sync custom integration sources
    jobs:
      - unit-tests
      - datahub-ingest:
          name: glean-source
          recipe: recipes/glean_recipe.dhub.yaml
          requires:
            - unit-tests
      - datahub-ingest:
          name: legacytelemetry-source
          recipe: recipes/legacy_recipe.dhub.yaml
          requires:
            - unit-tests
```

---

Makes sense to me. Curious what does the `name` parameter do in the workflow? I don't see it used in the job def'n